<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>168</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Browse results</string>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This palette allows you to step back and forth through search results. Use the &lt;span style=&quot; font-weight:600;&quot;&gt;Scratchpad&lt;/span&gt; to make lists of examples of special interest. The &lt;span style=&quot; font-weight:600;&quot;&gt;Discard&lt;/span&gt; button will remove resuts from the Results field on the Search tab.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
     border: 0px solid black;
     border-radius: 4px;
     min-width: 50px;
	 min-height: 18px;
     max-width: 76px;
	 max-height: 24px;
	 background: #6698FF;
	 color: white;
}

QPushButton:pressed {
     border: 2px solid black;
     border-radius: 8px;
}

</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QToolButton" name="PrevBtn">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Go backwards.</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="NextBtn">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>15</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Go forwards.</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>56</y>
     <width>218</width>
     <height>40</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>90</y>
     <width>226</width>
     <height>67</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Builds a list of earmarked examples from the search results
that are especially interesting or that you want to set aside for
some purpose or another. Scratchpads can be saved to a file
or converted into a Dataset.</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
    background: transparent;
     min-width: 32px;
	 min-height: 32px;
     max-width: 32px;
	 max-height: 32px;
}

QToolButton:pressed {
	border: 3px red;
}</string>
   </property>
   <property name="title">
    <string>Scratchpad</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="Select">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="toolTip">
       <string>Add this example to scratchpad.</string>
      </property>
      <property name="text">
       <string>Add to pad</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="NewList">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="toolTip">
       <string>Save current earmarked examples and start a new scratchpad.</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>New pad</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Save">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="toolTip">
       <string>Save earmarked examples.</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Discard">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>22</y>
     <width>76</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Remove this example from search results.</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Discard</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
